lib/registrant.rb

```diff
@@ -0,0 +1,21 @@
+class Registrant
+  attr_reader :name, 
+              :age, 
+              :license_data, 
+              :permit
+
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @license_data = {:written=>false, :license=>false, :renewed=>false}
+    @permit = permit
+  end
```

Comment:
This looks great. I like that you are using a default value for `permit` upon initialization.